docs/user/ovirt/install_upi.md

```diff
@@ -378,22 +385,22 @@ $ openshift-install create ignition-configs --dir ./wrk
 Other than the ignition files the installer generated
 
 - `auth` folder containing the admin credentials necessary to
-  connect to the cluster via the `oc` or `kubectl` CLIs.
-- `metadata.json` with informations like the ocp cluster name, ocp cluster ID and the `infraID`
-  (generated for the current running installation)
+  connect to the cluster via the `oc` or `kubectl` CLI utilities.
+- `metadata.json` with information like the OCP cluster name, OSC cluster ID and the `infraID`
```

Comment:
Just a little typo in the correction of my many typos:
- can you change *OSC* into *OCP*

Reply:
Sure :-D

Reply:
Done